schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts SMALLINT,
     neutered BOOLEAN,
-    weight_kg NUMERIC(6, 2) NOT NULL
+    weight_kg NUMERIC(6, 2) NOT NULL,
+    species VARCHAR(255)
```

Comment:
To change a table, you need to use the `ALTER` command. Something like this should do the trick: 
``` 
   ALTER TABLE animals ADD species varchar(255);
```